cime_config/SystemTests/rxcropmaturity.py

```diff
@@ -416,6 +454,30 @@ def _run_generate_gdds(self, case_gddgen):
             raise RuntimeError(error_message)
         self._gdds_file = generated_gdd_files[0]
 
+    def _run_interpolate_gdds(self):
+        # Save where?
```

Comment:
Is "Save where?" a statement of what's going on or a question of where it should save to? Could this be made clearer?

Reply:
The former; will clarify.

Reply:
Resolved in 787c2f2.